Code/SPActions.ts

```diff
@@ -23,13 +23,21 @@ module CSREditor {
                 if (window["WPQ" + wpqId + "FormCtx"]) {
```

Comment:
This covers only List Forms. Thus for List Views and Quick Edit template will not contain any field templates.